decidim-core/lib/decidim/core.rb

```diff
@@ -107,6 +107,10 @@ module Decidim
   autoload :AttributeObject, "decidim/attribute_object"
   autoload :Query, "decidim/query"
   autoload :Command, "decidim/command"
+  autoload :EventRecorder, "decidim/event_recorder"
+  autoload :NullQuery, "decidim/null_query"
+  autoload :SqlQuery, "decidim/sql_query"
```

Comment:
By the way, are we using the `SqlQuery` somewhere in Decidim core?

If not, I don't see necessary bringing it to the core.